docs/user-guide/sharding.html

```diff
@@ -392,14 +393,16 @@ <h3 id="shard-names-in-range-based-keyspaces">Shard Names in Range-Based Keyspac
 beginning with [ 0x80 ] and ending, noninclusively, with [ 0xc0], then
 the shard&#39;s name is <code>80-c0</code>.</p>
 
-<p>Using this naming convention, the following four shards would be a valid
-full partition:</p>
+<p>Using this naming convention, the following six shards would be a valid
```

Comment:
How about keeping the original example as a simple case and mention your advanced example right after that?

People are probably looking for a simple example first: A setup with 2 or 4 shards.

You can introduce your advanced example by stressing that the user can use more byte values to further divide the keyrange.

One other note: You're introducing the term "uneven". How about explaining "even" first right after the simple example?

Something like this:

> This is an example for a partition with even shards i.e. equally sized key ranges.

And then you can say that uneven shards are supported and more byte values as well :)